### ISSUE TITLE
Rsync automated image and kernel updates for CaSSIS

### ISSUE THREAD

**Issue by krlberry** [DOI-USGS/ISIS3#2638]
Add daily automated kernel and images updates from CaSSIS's remote server to `/work/projects/tgo` using rsync. 

This is waiting on authentication issues with the remote server, so cannot be completed at this time.

**Comment by jlaura** [comment]
This is currently waiting on input from external collaborators.

**Comment by krlberry** [comment]
Rsync is from the remote server is now set up, and this ticket could be worked on. In the unlikely case that someone besides me works on this, contact me for the server info and rsync command to get the files that Laz wants.

**Comment by jlaura** [comment]
@kberryUSGS @blandoplanet I am going to tag this 3.7.0 and we should then prioritize this along with the other products stuff.

**Comment by jlaura** [comment]
@kberryUSGS Can we close this as done?

**Comment by krlberry** [comment]
Need to update the permissions on the kernels that come down as part of this script. By default they come down with permissions that don't allow users to download.

**Comment by jlaura** [comment]
@kberryUSGS We either need to close this issue fixed today or remove the 3.7.0 label and stop working on it during the CSM sprint.

**Comment by krlberry** [comment]
@jlaura I am okay with removing the 3.7.0 tag. I haven't really worked on it during the CSM sprint. I just took care of Laz's rsync error from yesterday and updated this ticket with more info.

**Comment by ascbot** [comment]
I am a bot that cleans up old issues that do not have activity.

Happy Birthday to this issue! :birthday:

Unfortunately, this issue has not received much attention in the last 12 months. Therefore, I am going to close it. Please feel free to reopen this issue or open a new issue sometime in the future. If this issue is a bug report, please check that the issue still exists in our newest version before reopening.

**Comment by krlberry** [comment]
This was automatically closed, but the issue wasn't actually fixed until now. 

I'm going to remove the automatically_closed label since the issue was fixed, and to me "automatically_closed" implies "potentially not resolved." This issue was fixed by PR #12 into the internal scripts repo.
